robolectric-shadows/shadows-core/src/main/java/org/robolectric/fakes/RoboMenu.java

```diff
@@ -52,7 +51,7 @@ public MenuItem add(int groupId, int itemId, int order, CharSequence title) {
 
   @Override
   public MenuItem add(int groupId, int itemId, int order, int titleRes) {
-    return add(groupId, itemId, order, context.getResources().getString(titleRes));
+    return add(groupId, itemId, order, RuntimeEnvironment.application.getResources().getString(titleRes));
```

Comment:
You should use the context passed in then, it should work, unless you give me a reason why not.

#2 should probably also use a passed in context.